projects/components/src/navigation/nav-item/nav-item.component.ts

```diff
@@ -11,7 +11,7 @@ import { NavItemLinkConfig } from '../navigation-list.component';
   template: `
     <ht-link *ngIf="this.config" [paramsOrUrl]="buildNavigationParam | htMemoize: this.config">
       <div
-        *htIfFeature="this.config.features | htFeature as featureState"
+        *htIfFeature="this.config.featureState$ | async as featureState"
```

Comment:
ooh.. @itssharmasandeep I think we still need to apply `htFeature` here

Reply:
Nope

Reply:
`htFeature` takes the `features` which are `string[]` and convert this to `featureState`
I am converting this to `featureState` in the service itself and pass here